### DISCUSSION TITLE
Colum filter performance

### DISCUSSION THREAD

**Discussion by lostluna1** [w-ahmad/WinUI.TableView#60] [Feedback Wanted]
Regarding the performance issue of header filtering, if my dataset is at the 10K level, it takes several seconds for the UI to respond after clicking on the filter. Personally, I believe that column header filtering exists because I am too lazy to do pagination

**Comment by w-ahmad** [comment]
The reason it is slow is because there is `ItemsControl` hosted in `ScrollViewer` to show filter items without virtualization. Replacing these controls with a `ListView` could significantly enhance the performance of the filter flyout.

**Comment by w-ahmad** [comment]
created bug from this discussion #65
